system-tests/src/test/java/org/eclipse/edc/samples/transfer/ConsumerPullSampleTestCommon.java

```diff
@@ -0,0 +1,222 @@
+                "        \"edc\": \"https://w3id.org/edc/v0.0.1/ns/\",\n" +
+                "        \"odrl\": \"http://www.w3.org/ns/odrl/2/\"\n" +
+                "    },\n" +
+                "    \"@id\": \"aPolicy\",\n" +
+                "    \"policy\": {\n" +
+                "        \"@type\": \"set\",\n" +
+                "        \"odrl:permission\": [],\n" +
+                "        \"odrl:prohibition\": [],\n" +
+                "        \"odrl:obligation\": []\n" +
+                "    }\n" +
+                "}";
+
+        given().headers("Content-Type","application/json")
+                .contentType(ContentType.JSON)
+                .body(requestBody)
+                .when()
+                .post(MANAGEMENT_API_URL+"/management/v2/assets")
+                .then()
+                .statusCode(HttpStatus.SC_OK)
+                .body("@id", not(emptyString()));
+    }
+
+    public void createPolicyDefinition() {
+        String requestBody = "{\n" +
+                "    \"@context\": {\n" +
+                "        \"edc\": \"https://w3id.org/edc/v0.0.1/ns/\",\n" +
+                "        \"odrl\": \"http://www.w3.org/ns/odrl/2/\"\n" +
+                "    },\n" +
+                "    \"@id\": \"aPolicy\",\n" +
+                "    \"policy\": {\n" +
+                "        \"@type\": \"set\",\n" +
+                "        \"odrl:permission\": [],\n" +
+                "        \"odrl:prohibition\": [],\n" +
+                "        \"odrl:obligation\": []\n" +
+                "    }\n" +
+                "}";
+
+        given()
+                .header("Content-Type", "application/json")
+                .contentType(ContentType.JSON)
+                .body(requestBody)
+                .when()
+                .post(MANAGEMENT_API_URL + "/management/v2/policydefinitions")
+                .then()
+                .statusCode(HttpStatus.SC_OK)
+                .body("@id", not(emptyString()));
+    }
+
+    public void createContractDefinition(){
+        String requestBody = "{\n" +
+                "    \"@context\": {\n" +
+                "        \"edc\": \"https://w3id.org/edc/v0.0.1/ns/\"\n" +
+                "    },\n" +
+                "    \"@id\": \"1\",\n" +
+                "    \"accessPolicyId\": \"aPolicy\",\n" +
+                "    \"contractPolicyId\": \"aPolicy\",\n" +
+                "    \"assetsSelector\": []\n" +
+                "}";
+
+        ContractOfferId=given()
+                .header("Content-Type", "application/json")
+                .contentType(ContentType.JSON)
+                .body(requestBody)
+                .when()
+                .post(MANAGEMENT_API_URL + "/management/v2/contractdefinitions")
+                .then()
+                .statusCode(HttpStatus.SC_OK)
+                .body("@id", not(emptyString()))
+                .extract()
+                .jsonPath()
+                .get("@id");
+    }
+
+    public void fetchCatalog(){
```

Comment:
```suggestion
    public String fetchCatalog(){
```
See [comment](https://github.com/FraunhoferISST/edc-samples/pull/1#discussion_r1328359276).